mix.exs

```diff
@@ -80,7 +80,8 @@ defmodule Screens.MixProject do
       {:recon, "~> 2.5.6"},
       {:remote_ip, "~> 1.2"},
       {:retry, "~> 0.18.0"},
-      {:screens_config, github: "mbta/screens-config-lib"},
+      {:screens_config,
+       github: "mbta/screens-config-lib", branch: "rs/wayfinding-only-departures-section"},
```

Comment:
Noting that this will of course need to be modified back before merging, once the `screens-config-lib` PR is merged. This is needed for testing currently though.